modules/claims_api/app/swagger/claims_api/v1/swagger.json

```diff
@@ -497,7 +497,7 @@
               "application/json": {
```

Comment:
Definitely not a big deal, but anytime I make changes to V2 where I have to also regenerate the swagger docs, I just discard the change to the V1 swagger docs.  Since nothing has really changed in the V1 swagger doc as part of this PR, you'd be ok just discarding it.